apps/testing-javascript/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -1,24 +1,24 @@
 import SkillRecordings, {
   type SkillRecordingsOptions,
-  defaultPaymentOptions,
-  StripeProvider,
+  // defaultPaymentOptions,
+  // StripeProvider,
 } from '@skillrecordings/skill-api'
 import {nextAuthOptions} from '../auth/[...nextauth]'
 
-const paymentOptions = defaultPaymentOptions({
-  stripeProvider: StripeProvider({
-    stripeSecretKey: process.env.STRIPE_SECRET_TOKEN,
-    apiVersion: '2020-08-27',
-  }),
-})
```

Comment:
why is this being commented out?

this is the direction we're heading to support multi-provider payments.

Reply:
![CleanShot 2024-03-01 at 17 35 11@2x](https://github.com/skillrecordings/products/assets/1519448/639805bf-c5e5-4910-840a-e89795cdb225)

Reply:
@jbranchaud I think that correct import from is `'@skillrecordings/skill-api/src'` and not `'@skillrecordings/skill-api/`. Can you confirm?

Reply:
pushed the fix

Reply:
now build fails on this:
![CleanShot 2024-03-01 at 18 51 19@2x](https://github.com/skillrecordings/products/assets/1519448/cd3d8ba3-a2a9-4bcb-a8a8-7823b533a00a)

Reply:
The import should have already been correct. I think it may be an issue of re-building `skill-api` locally (`cd packages/skill-api; pnpm dev`).

Reply:
![CleanShot 2024-03-01 at 20 39 18@2x](https://github.com/skillrecordings/products/assets/1519448/6f62c4de-1d6a-43e8-ab06-34f95408c938)

Reply:
can't build on my side. Does it work on yours?